core/src/main/resources/lib/layout/svgIcon.jelly

```diff
@@ -0,0 +1,49 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:d="jelly:define">
+    <st:documentation>
+        <!-- TODO: @since? -->
```

Comment:
I don't think we put those on `jelly` files.

Reply:
It's used on about 7-8 jelly templates. This is one example:   https://github.com/jenkinsci/jenkins/blob/dc8781dd69dc6fed1b95ea0837e5aa74982c8975/core/src/main/resources/lib/layout/css.jelly#L30 . I'd rather keep it.